skfda/exploratory/visualization/_outliergram.py

```diff
@@ -50,22 +63,39 @@ class Outliergram:
     def __init__(
         self,
         fdata: FDataGrid,
+        chart: Union[Figure, Axes, None] = None,
+        *,
+        fig: Optional[Figure] = None,
+        axes: Optional[Axes] = None,
+        n_rows: Optional[int] = None,
+        n_cols: Optional[int] = None,
+        **kwargs,
     ) -> None:
+        BasePlot.__init__(self)
         self.fdata = fdata
         self.depth = ModifiedBandDepth()
         self.depth.fit(fdata)
         self.mbd = self.depth(fdata)
         self.mei = self.modified_epigraph_index_list()
+        if self.mbd.size != self.mei.size:
```

Comment:
I think `len` is more clear than `size` (the latter wouldn't work if the numpy array had more than one dimension.